packages/autocomplete/src/component/Autocomplete.jsx

```diff
@@ -132,11 +145,22 @@ class Autocomplete extends Component<Props, IState> {
     this.searchMode.search(value).then((results: Array<Item>) => {
       this.setState({
         results,
-        cursor: 0
+        cursor: this.props.allowNewEntry ? 0 : 1
```

Comment:
Why not just use booleans to do this?

Reply:
This is to select the start position of the cursor, not having to do with booleans.